playbooks/roles/ocp-upgrade/tasks/eus_upgrades.yaml

```diff
@@ -144,6 +170,19 @@
     msg: "Master MachineConfigPool not get updated"
   when: master_mcp.resources[0].status.updatedMachineCount != master_mcp.resources[0].status.machineCount
 
+- name: Get kubelet version of workers after final upgrade
+  shell: oc get node -l node-role.kubernetes.io/worker -o json | jq ".items[].status.nodeInfo.kubeletVersion"
```

Comment:
why are we using oc cli here instead of using the info module?

Reply:
Now using info module